proxy-node-gateway/src/main/java/uk/gov/ida/notification/resources/HubResponseResource.java

```diff
@@ -42,7 +43,7 @@ public HubResponseResource(
     @Path(Urls.GatewayUrls.GATEWAY_HUB_RESPONSE_PATH)
     @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
     public View hubResponse(
-        @FormParam(SamlFormMessageType.SAML_RESPONSE) String hubResponse,
+        @FormParam(SamlFormMessageType.SAML_RESPONSE) @ValidBase64SamlResponse String hubResponse,
```

Comment:
Please see above comment about saml parsing.

Reply:
Removed and replaced with a simple Base64 check.